### PR TITLE
Manage evidence note filter changes

### PR DESCRIPTION
**Pull request by sreedhar-civica**

### CODE REVIEW

File: src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs [hunk 1]
```diff
@@ -133,13 +135,13 @@ public async void HandleAsync_GivenRequestWithSearchRef_EvidenceDataAccessShould
         {
             //arrange
             var searchRef = TestFixture.Create<string>();
-            var request = GetAatfNotesRequest(SystemTime.UtcNow.Year, searchRef);
+            var wasteTypeList = TestFixture.CreateMany<WasteType>(2).ToList();
```
Comment (JonnyRush) [inline src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs:138]: Magic string (well int but yno) here, better to use the enum?

File: src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs [hunk 2]
```diff
@@ -155,13 +157,13 @@ public async void HandleAsync_GivenRequestWithRecipientFilterSet_EvidenceDataAcc
         {
             //arrange
             Guid? recipientId = Guid.NewGuid();
-            var request = GetAatfNotesRequest(SystemTime.UtcNow.Year, null, recipientId);
+            var wasteTypeList = TestFixture.CreateMany<WasteType>(2).ToList();
```
Comment (JonnyRush) [inline src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs:160]: Same magic int comment as above

File: src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs [hunk 3]
```diff
@@ -175,23 +177,23 @@ public async void HandleAsync_GivenRequestWithRecipientFilterSet_EvidenceDataAcc
         [Fact]
         public async void HandleAsync_GivenRequestWithWasteTypeFilterSet_EvidenceDataAccessShouldBeCalledOnce()
         {
-            //arrange
-            WasteType? wasteType = TestFixture.Create<WasteType?>();
-            var request = GetAatfNotesRequest(currentYear, null, null, wasteType);
+            //arrange            
+            var wasteTypeList = TestFixture.CreateMany<WasteType>(2).ToList();
```
Comment (JonnyRush) [inline src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs:181]: Magic int

File: src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs [hunk 4]
```diff
@@ -175,23 +177,23 @@ public async void HandleAsync_GivenRequestWithRecipientFilterSet_EvidenceDataAcc
...
             // assert
             A.CallTo(() => noteDataAccess.GetAllNotes(A<NoteFilter>.That.Matches(e =>
                 e.OrganisationId.Equals(request.OrganisationId) &&
                 e.AatfId.Equals(request.AatfId) &&
                 e.AllowedStatuses.SequenceEqual(status) &&
-                e.SearchRef == null && 
-                e.RecipientId == null && e.WasteTypeId.Equals((int?)wasteType) &&
+                e.SearchRef == null &&
+                e.RecipientId == null &&
+                //e.WasteTypeId == 1 &&
```
Comment (JonnyRush) [inline src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs:196]: Commented filter could be removed

File: src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs [hunk 5]
```diff
@@ -201,7 +203,8 @@ public async void HandleAsync_GivenRequestWithNoteStatusFilterSet_EvidenceDataAc
         {
             //arrange
             NoteStatus? noteStatus = TestFixture.Create<NoteStatus?>();
-            var request = GetAatfNotesRequest(currentYear, null, null, null, noteStatus);
+            var wasteTypeList = TestFixture.CreateMany<WasteType>(2).ToList();
```
Comment (JonnyRush) [inline src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs:206]: Magic Int

File: src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs [hunk 6]
```diff
@@ -225,7 +228,8 @@ public async void HandleAsync_GivenRequestWithStartDateFilterSet_EvidenceDataAcc
         {
             //arrange
             var startDate = TestFixture.Create<DateTime?>();
-            var request = GetAatfNotesRequest(currentYear, null, null, null, null, startDate);
+            var wasteTypeList = TestFixture.CreateMany<WasteType>(2).ToList();
```
Comment (JonnyRush) [inline src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs:231]: Magic int

File: src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/Overview/ViewAllOtherEvidenceOverview.cshtml [hunk 7]
```diff
@@ -145,27 +149,45 @@
...
+                                    endDate = Model.ManageEvidenceNoteViewModel.SubmittedDatesFilterViewModel.EndDate.HasValue ? Model.ManageEvidenceNoteViewModel.SubmittedDatesFilterViewModel.EndDate.Value.ToShortDateString() : null,
+                                    receivedId = Model.ManageEvidenceNoteViewModel.RecipientWasteStatusFilterViewModel.ReceivedId,
+                                    wasteTypeValue = Model.ManageEvidenceNoteViewModel.RecipientWasteStatusFilterViewModel.WasteTypeValue.HasValue ? Model.ManageEvidenceNoteViewModel.RecipientWasteStatusFilterViewModel.WasteTypeValue.Value.ToInt() : (int?)null,
+                                    noteStatusValue = Model.ManageEvidenceNoteViewModel.RecipientWasteStatusFilterViewModel.NoteStatusValue.HasValue ? Model.ManageEvidenceNoteViewModel.RecipientWasteStatusFilterViewModel.NoteStatusValue.Value.ToInt() : (int?)null,
+                                    submittedBy = Model.ManageEvidenceNoteViewModel.RecipientWasteStatusFilterViewModel.SubmittedBy,
                                 }
                             }
                         });
                     }
+
```
[the remark this answers is not in the document]
Comment (sreedhar-civica) [reply src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/Overview/ViewAllOtherEvidenceOverview.cshtml:168]: Removed commented code.

File: src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs (hunk 4 above)
Comment (sreedhar-civica) [reply src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs:196]: Removed commented code

File: src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs (hunk 1 above)
Comment (sreedhar-civica) [reply src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs:138]: Changed code as suggested

File: src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs (hunk 2 above)
Comment (sreedhar-civica) [reply src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs:160]: Changed code as suggested

File: src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs (hunk 3 above)
Comment (sreedhar-civica) [reply src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs:181]: Changed code as suggested

File: src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs (hunk 5 above)
Comment (sreedhar-civica) [reply src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs:206]: Changed code as suggested

File: src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs (hunk 6 above)
Comment (sreedhar-civica) [reply src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfNotesRequestHandlerTests.cs:231]: Changed code as suggested

**Review by JonnyRush — APPROVED** [review]

**Pull request merged**